### PR TITLE
fixed relationships and scraper

### PR DESCRIPTION
**Pull request by cchchoi1986**

### CODE REVIEW

**Comment by cchchoi1986** [comment]
@harrychenca @paulld

File: app/assets/javascripts/plants.js.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (h8rry) [inline app/assets/javascripts/plants.js.coffee]: delete all empty files

File: app/assets/javascripts/trails.js.coffee [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (h8rry) [inline app/assets/javascripts/trails.js.coffee]: delete all empty files

File: app/assets/stylesheets/plants.css.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the plants controller here.
```
Comment (h8rry) [inline app/assets/stylesheets/plants.css.scss]: delete all empty files

File: app/assets/stylesheets/trails.css.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the trails controller here.
```
Comment (h8rry) [inline app/assets/stylesheets/trails.css.scss]: delete all empty files

File: app/helpers/plants_helper.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+module PlantsHelper
```
Comment (h8rry) [inline app/helpers/plants_helper.rb]: delete all empty files

File: app/helpers/trails_helper.rb [hunk 6]
```diff
@@ -0,0 +1,2 @@
+module TrailsHelper
```
Comment (h8rry) [inline app/helpers/trails_helper.rb]: delete all empty files

File: lib/tasks/make_region_trails.rake [hunk 7]
```diff
@@ -0,0 +1,8 @@
+namespace :make_region_trails do
+  desc "link region to trails of hong kong"
+  task :create => :environment do
+
```
Comment (h8rry) [inline lib/tasks/make_region_trails.rake:4]: why empty

Comment (cchchoi1986) [inline lib/tasks/make_region_trails.rake:4]: for another branch

**Pull request merged**